.github/CHANGELOG.md

```diff
@@ -8,46 +8,47 @@
+8. [HYD] Added hydraulic reverser actuators - @Crocket63 (crocket)
+9. [SD] Added a display duration of 3s for STS page - @pluce
+10. [EIS] Added > character to the font - @KiloEchoVictor (Kevin)
+11. [ND] Mask the map below the TCAS/WXR and FM messages - @tracernz (Mike)
+12. [FMS] Improved robustness of ILS selection - @tracernz (Mike)
+13. [MCDU] The remote MCDU protocol now sends the annunciator light state - @tracernz (Mike)
+14. [MCDU] The remote MCDU protocol now sends the display and integral light brightness - @tracernz (Mike)
+15. [MCDU] The remote MCDU protocol now supports the BRT/DIM keys - @tracernz (Mike)
+16. [PFD] Corrected the NO DH logic in the FMS and PFD - @tracernz (Mike)
+17. [FDR] Add secondary engine parameters to FDR - @Taz5150 (TazX [Z+2]#0405)
+18. [FLIGHTMODEL] Fix outer to inner tank transfer valve closure - @donstim (donbikes)
+19. [SD] Corrected thickness and position of lower ECAM fuel page lines - @robertxing2004 (robeet)
+20. [SD] Corrected position of outer tank fuel quantity values - @robertxing2004 (robeet)
+21. [ISIS] Fixed the ISIS alt tape order for negative altitudes - @aweissoertel (Alex)
+22. [MCDU] Add basic annunciator support - @tracernz (Mike)
+23. [MCDU] Add basic MCDU MENU functionality - @tracernz (Mike)
+24. [MCDU] Split subsystem scratchpads - @tracernz (Mike)
+25. [APU] APU can now consume fuel - @tracernz (Mike)
+26. [EFB] Added UI message on fuel/payload page when GSX is activated - @frankkopp (Frank Kopp)
+27. [FWC] WING TANK LO LVL message now considers centre tank mode selector - @tracernz (Mike)
+28. [MCDU] PERF APPR page no longer demands QNH and TEMP when outside 180NM from destination - @robertxing2004 (robeet)
+29. [FMS] Add basic support for FA legs - @tracernz (Mike)
+30. [MCDU] Don't calculate V-speeds when FoB not available - @tracernz (Mike)
+31. [MCDU] ZFW/ZFWCG auto-fill uses desired values when boarding in progress - @tracernz (Mike)
+32. [EFB/FWC] Auto Call Outs selectable - @tracernz (Mike)
+33. [FWC] Added "twenty five hundred" and "two thousand" callouts - @tracernz (Mike)
+34. [FMS] Load procedures that reference a runway correctly - @tracernz (Mike)
+35. [MCDU/ND] Corrected altitude formatting with respect to transition altitude and transition level - @tracernz (Mike)
+36. [PFD] Adjusted size of pitch trim messages - @tracernz (Mike)
+37. [BLEED] Allow engine start from crossbleed air - @BlueberryKing (BlueberryKing)
+38. [BLEED] Improve bleed page visuals on the SD - @BlueberryKing (BlueberryKing)
+39. [BLEED] Use more accurate pneumatic valves and sensors - @BlueberryKing (BlueberryKing)
+40. [EFB] flyPad OS 3.2 - Cosmetic changes for throttle configuration settings and support for multiple airframes - @2hwk (2cas)
+41. [SD] Connect SD BLEED page APU bleed pressure indication to correct ADRs - @BlueberryKing (BlueberryKing)
+42. [EFB] Change the order of the weather dashboard elements to reflect the order of the MCDU APPR PERF phase - @oim (oim1)
```

Comment:
Please keep all these 1.... markdown handles the numbering automatically. Also, move your own one to v0.12.0 (after a rebase/merge of master).